File: Python/Conditional_Statements/Leap_Year_or_not.py
year = int(input("Enter the year you want to check:"))
if(year%100==0):
    if(year%400==0):
        print("This is Leap year!")
    else:
        print("This is not Leap year!")
elif(year%4==0):
    print("This is Leap year!")
else:
    print("This is not Leap year!")

File: C/Conditional Instructions/leap_year_or_not.c
#include <stdio.h>
#include <conio.h>
int main()
{
    int year;
    printf("Enter the Year :");
    scanf("%d", &year);
    if (year % 100 == 0 && year % 400 == 0)
    {
        printf("The year is leap year");
    }
    else if (year % 4 == 0)
    {
        printf("The year is leap year");
    }
    else
    {
        printf("The year is not leap year");
    }
    return 0;
}